api/WorkerGlobalScope.json

```diff
@@ -126,6 +126,47 @@
           }
         }
       },
+      "fonts": {
```

Comment:
This is the new accessor for the API in Workers. I mirrored the Chrome support for this API in other docs.